nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -1015,8 +1022,8 @@ define(
                         title: 'Error initializing app state',
                         message: ex.message,
                         advice: [
-                            'Reset the app with the red recycle button and try again.',
-                            'If that fails, delete the app cell and re-insert it.',
```

Comment:
not sure how long it is since there was a red recycle button instead of a `reset` button...